DEPLOYMENT.md

```diff
@@ -0,0 +1,57 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wild card SSL certificate be installed on the OpenShift router. However, in situations where this is not desirable, RHMAP can be exposed using single URL.
+
+## RHMAP Proxy
```

Comment:
Do we want to name this `RHMAP Proxy`? Current project name is `wildcard proxy`
I'm open for suggestions